### PR TITLE
Simon aav 2

### PR DESCRIPTION
**Pull request by simonleandergrimm**
Added an estimate for AAV-2. Data is quite limited, I based the US estimate on a study that estimates seroprevalence among hemophilic patients from different nations.

### CODE REVIEW

**Comment by jeffkaufman** [comment]
This change seems to include your EBV changes?  There are two ways to handle this:

1. Wait to send this out for review until you've merged your EBV changes to main.
2. Reconfigure this PR as a proposed merge onto the EBV branch, then after merging EBV to main this PR will automatically switch to be a proposed merge onto main.

Usually (2) is better, though it's a bit more complex.

**Comment by simonleandergrimm** [comment]
> This change seems to include your EBV changes? There are two ways to handle this:
> 
> 1. Wait to send this out for review until you've merged your EBV changes to main.
> 2. Reconfigure this PR as a proposed merge onto the EBV branch, then after merging EBV to main this PR will automatically switch to be a proposed merge onto main.
> 
> Usually (2) is better, though it's a bit more complex.

Fixed! Picked (2)

**Review by jeffkaufman — APPROVED** [review]

File: pathogens/aav2.py [hunk 1]
```diff
@@ -3,27 +3,59 @@
...
+)
+
 
 def estimate_prevalences() -> list[Prevalence]:
-    stub_estimate = Prevalence(
-        # Completely made up, checking in stub for model testing.
-        infections_per_100k=50_000,
-        active=Active.LATENT,
-        country="United States",
+    # We assume that global seroprevalence will be similar to seroprevalence
```
Comment (jeffkaufman) [inline pathogens/aav2.py:39]: Consider also mentioning Figure 1, showing the range of seroprevalences by country?

Comment (simonleandergrimm) [reply pathogens/aav2.py:39]: done!

**Pull request merged**